packages/extension/src/view/devtools/components/stories/cookies/cookieDetails/index.tsx

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Could you tell me the reason for keeping these components inside stories? Are they not supposed to be real components that will be used in the devTools?

Reply:
I went ahead with idea that first all components will be made to work as story components and after UI designs gets approved these get integrated for devtools panel at later point.

I have integrated these components to devtools panel and also updated their location.

Reply:
I did not mean to remove all stories, I was suggesting that components shouldn't be inside the stories folder as they are supposed to be used in Developer Tools. We still want to show these components as stories.

Reply:
> I have integrated these components to devtools panel and also updated their location.

👍